ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -68,6 +68,15 @@ def Cosh(*, x, name="cosh"):
     return ivy.cosh(x)
 
 
+def Cumprod(*, x, axis, exclusive=False, reverse=False, name=None):
+    if reverse:
```

Comment:
The `ivy.cumprod` function has just been updated to handle the `reverse` argument, could you please use this here instead? 😄 

Thanks!